doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Configure MQTT (Client)"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a PubSub Integration using the built in MQTT client and an external MQTT server.
+
+<!--more-->
+
+Creating a PubSub integration with the built in MQTT Client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client should work with any MQTT server.
+
+In your application select the **Pub-subs** submenu from the **Integrations** side menu. Clicking on the **+ Add pub-sub** button will open the PubSub creation screen.
+
+Give your PubSub an ID, choose a format, and enter a **Base Topic** to subscribe or publish to.
+
+Select **MQTT** as provider.
+
+{{< figure src="pubsub-id.png" alt="Pubsub creation screen" >}}
+
+If using Client Certificates, enable **Secure**. You will be asked to upload the **Server root CA certificate**, the **Client certificate**, and the **Client private key.**
```

Comment:
"Root CA certificate"

Inconsistent casing "Base Topic" (title case) vs "Client private key" (sentence case). Is this literally how it is in the Console?

Reply:
This is how it is in the console. Should they be title case?

<img width="638" alt="Screenshot 2020-05-12 at 17 36 07" src="https://user-images.githubusercontent.com/6963436/81714397-4f428500-9477-11ea-969e-7b6852134950.png">
<img width="647" alt="Screenshot 2020-05-12 at 17 36 14" src="https://user-images.githubusercontent.com/6963436/81714404-523d7580-9477-11ea-9631-37d93720b978.png">
<img width="326" alt="Screenshot 2020-05-12 at 17 37 08" src="https://user-images.githubusercontent.com/6963436/81714406-52d60c00-9477-11ea-8662-f946fd9fd9be.png">

Reply:
We moved to sentence case in the Console recently, as per the design guidelines. If there is title case somewhere then that needs to be fixed.